doc/Extensions/Applications.md

```diff
@@ -1146,3 +1146,21 @@ echo "extend zfs /etc/snmp/zfs-linux" >> /etc/snmp/snmpd.conf
 ```
 
 Now restart snmpd and you're all set.
+
+###### CentOS 7
```

Comment:
This is a bit much. Don't go into details of how to configured Python here. Instead list the dependancies and let the users deal with the rest.